### ISSUE TITLE
[FEAT]: Buttons Next \ Previous file or episode from given directory

### ISSUE THREAD

**Issue by odminik** [nova-video-player/aos-AVP#1274]
### Description

Hello all. 

I was wondering if it would be possible to add more controls. In this case Next / Previous file/episode from the given directory. 

I found issue like: https://github.com/nova-video-player/aos-AVP/issues/443,  but it looks like it is forgotten? Are there any chances for reviewing it?

Explanation:

It will be comfortable don't need to wait till the end of a given file and switch to the next episode or movie in the given directory, without exiting the folder menu. 

Also going back a few episodes without going to the menu would be useful. 

And I would like to thank you all who work to make better this awesome player :)

Thank you

**Comment by j0vand** [comment]
I completely agree with your statement. I also think this feature is crucial for my daily use. I hope the author can consider adding this functionality. Thank you!
